package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
Nope, looks like I was right - https://github.com/addaleax/lzma-native/issues/30#issuecomment-287521987 and https://github.com/resin-io/etcher/pull/1191